packages/editor-ui/src/components/CredentialEdit/CredentialDocs.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+	<div v-if="docs" :class="$style.docs">
```

Comment:
Can you move this v-if directive to have it on the component in the parent component so it doesn't render this component at all if there is no docs?

And you can pass the docs as prop

Like `<CredentialDocs v-if="docs"` :docs="docs" />